config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config and
+# CERTIFICATE_NAMESPACE and CERTIFICATE_NAME will be substituted by kustomize
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/name: validatingwebhookconfiguration
+    app.kubernetes.io/instance: validating-webhook-configuration
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: siteconfig
+    app.kubernetes.io/part-of: siteconfig
+    app.kubernetes.io/managed-by: kustomize
+  name: validating-webhook-configuration
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
```

Comment:
I think instead of doing this here you'll need to have all of this directly in the manifest.

This is because of the issue I mentioned before where the ACM installer doesn't know how to render files using kustomize or create a webhook properly based on what's in the CSV.

Essentially you need to make sure you have a yaml somewhere for the webhook and service that can be directly ingested by the installer automation and used without kustomize involved.